Low quality image placeholder for images (#182)
torchbox/wagtail-grapple: docs/general-usage/usage-with-frontend-frameworks.rst

Usage with frontend frameworks

> ##################################

Reviewer: ```suggestion
##############################
```

> A few features of Wagtail Grapple are useful for specific frontend frameworks.

Reviewer: ```suggestion
Several Wagtail Grapple are useful for specific frontend frameworks.
```
